_posts/2020-06-22-snowplow-java-tracker-0.10.1/index.md

```diff
@@ -0,0 +1,132 @@
+
+<h2 id="updates">4. Dependency updates</h2>
+
+We have also bumped a number of dependencies in the Java Tracker in this release, bringing all of its dependencies up to date.
+
+Specifically, we have updated the following libraries:
+* Upgrade to Gradle 6 ([#236](https://github.com/snowplow/snowplow-java-tracker/issues/236))
+* Bump org.apache.httpcomponents:httpasyncclient to 4.1.4 ([#249](https://github.com/snowplow/snowplow-java-tracker/issues/249))
+* Bump org.apache.httpcomponents:httpclient to 4.5.12 ([#248](https://github.com/snowplow/snowplow-java-tracker/issues/248))
+* Bump mockito-core to 3.3.3 ([#247](https://github.com/snowplow/snowplow-java-tracker/issues/247))
+* Bump slf4j-api to 1.7.30 ([#246](https://github.com/snowplow/snowplow-java-tracker/issues/246))
+* Bump commons-net to 3.6 ([#245](https://github.com/snowplow/snowplow-java-tracker/issues/245))
+* Bump commons-codec to 1.14 ([#241](https://github.com/snowplow/snowplow-java-tracker/issues/241))
+* Bump mockwebserver to 4.7.2 ([#239](https://github.com/snowplow/snowplow-java-tracker/issues/239))
+* Bump guava to 29.0 ([#238](https://github.com/snowplow/snowplow-java-tracker/issues/238))
+* Bump wiremock to 2.26.3 ([#237](https://github.com/snowplow/snowplow-java-tracker/issues/237))
+* Bump jackson-databind to 2.11.0 ([#235](https://github.com/snowplow/snowplow-java-tracker/issues/235))
+
+<h2 id="changes">5. Breaking changes</h2>
+
+By updating the Java Tracker to handle sending events asynchronously, we have also introduced some breaking changes in this release. While these provide a cleaner API, they will cause compilation errors if you have instrumented a previous version.
+1. The signature on the callback for `requestCallback` on the Emitter has changed, so events which have failed to send return the `Event` object rather than the internal `TrackerPayload`. This allows you to easily resend them using the usual `Tracker.track(Event)` method.
+2. Given the above, we have removed `Tracker.track(TrackerPayload)` as you should never need to send the raw payload. If you do need to, you should rather extend `AbstractEvent`.
+3. Tracker parameters are now immutable once the Tracker has been constructed. This means functions such as `Tracker.setNamespace()` no longer exist. To set these parameters, you can utilize the `TrackerBuilder`. This change has been made as Payloads are now constructed asynchronously, so changing the Tracker parameters after initial construction may lead to unexpected results.
+
+<h2 id="upgrade">6. Upgrading</h2>
+
+The tracker is available on [Snowplow's Bintray Maven Repository](https://bintray.com/snowplow/snowplow-maven/snowplow-java-tracker) and [jcenter](https://bintray.com/bintray/jcenter?filterByPkgName=snowplow-java-tracker).
```

Comment:
From 0.9.0 it’s also available on Maven Central too. We should list that as Maven Central is the more popular repository. I’d drop the mention of jCenter to keep this cleaner.

Reply:
Does that look better?